ubuntudesign/www.ubuntu.com
# Add new IoT security takeunder

## Done

Add new IoT security takeunder
Amend MWC takeunder as per card instructions

## QA

Please check that the words in the takeunder match the words in the doc and that the small change to the MWC takunder corresponds to the change requested in the image that’s attached to the card.

## Issue / Card

Card: https://trello.com/c/nT3eeNjM
Doc: https://docs.google.com/document/d/1RX5QJxMQOVbUKNRTj_RHHziI13Xl9YuEBNxfCYN1O8s/edit#
